docs/src/pages.js

```diff
@@ -31,6 +31,7 @@ dataGridComponent.children = [
   },
   { pathname: '/components/data-grid/demo' },
   { pathname: '/components/data-grid/getting-started' },
+  { pathname: '/components/data-grid/migration-v4' },
```

Comment:
By default, the title is derived from the pathname.

```suggestion
  { pathname: '/components/data-grid/migration-v4', title: 'Migration From v4' },
```